### PR TITLE
Fix source maps for minified dist files

### PR DESCRIPTION
**Pull request by ericf**
This fixes the missing source content in minified source files as well as improves the source maps for this package overall.

### CODE REVIEW

**Comment by yahoocla** [comment]
CLA is valid!

**Comment by caridy** [comment]
looking good, I don't fully understand why we need to remove the `sourceRoot` from the maps though.

File: package.json [hunk 1]
```diff
@@ -47,13 +47,14 @@
     "grunt-contrib-connect": "^0.9.0",
     "grunt-contrib-copy": "^0.7.0",
     "grunt-contrib-jshint": "^0.10.0",
-    "grunt-contrib-uglify": "^0.6.0",
+    "grunt-contrib-uglify": "^0.7.0",
     "grunt-extract-cldr-data": "^1.1.0",
-    "grunt-saucelabs": "~8.3.2",
+    "grunt-json-remove-fields": "git://github.com/ericf/grunt-json-remove-fields#improve-files-and-output",
```
Comment (ericf) [inline package.json]: Waiting on https://github.com/diepet/grunt-json-remove-fields/pull/1 to make this a proper npm registry dep.

File: Gruntfile.js [hunk 2]
```diff
@@ -110,6 +121,16 @@ module.exports = function (grunt) {
             }
         },
 
+        json_remove_fields: {
+            min_source_maps: {
+                options: {
+                    fields: ['sourceRoot']
```
Comment (ericf) [inline Gruntfile.js]: This is a quick fix to get around `grunt-contrib-uglify` limitations. Adding a `sourceRoot` the `uglify` task config is required to get the source contents to actually end up in the map (fixing the issue this PR addresses). But this leads to `"intl-messageformat/intl-messageformat/"` for the source files in the browser dev tools, so this Grunt task removes the extra `"intl-messageformat/"` from the `.min.js.map` files.

Comment (caridy) [inline Gruntfile.js]: got it

Comment (ericf) [inline Gruntfile.js]: /cc @caridy

**Pull request merged**